contracts/StreamManager.sol

```diff
@@ -135,31 +129,17 @@ contract StreamManager is IStreamManager, ReentrancyGuard {
             _rate,
             _terminationPeriod,
             _cliffPeriod,
-            block.timestamp, // created at
```

Comment:
@accompany1205 why did you remove created at? because the struct still has it?

Reply:
Where's it?

Reply:
We don't need createdAt for now. If we need it in the future, I will add.